src/Hazelcast.Net/Exceptions/StackTraceElement.cs

```diff
@@ -12,7 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if NETSTANDARD2_1
+#if NETFRAMEWORK || NETSTANDARD2_0
+#else
```

Comment:
why have a empty if block?

Reply:
ah... because in the past it was not empty, but yes I could change this to:
```
#if !NETFRAMEWORK && !NETSTANDARD2_0
```
will do